databases/week2/create tables-ER diagram.sql

```diff
@@ -0,0 +1,41 @@
+-- Part 4-ER diagram -- creating tables
+
+CREATE TABLE `customers` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL,
+   `user_name` varchar(255) NULL
+) 
+
+CREATE TABLE `products` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` varchar(255) NOT NULL,
+  `price` int(10) unsigned NOT NULL
+)
+
+
+CREATE TABLE `orders` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `amount` int(10) NOT NULL,
+  `customer_id` int(10) unsigned NOT NULL
+   `product_id` int(10) unsigned NOT NULL
```

Comment:
this model will be able to save only one product per order.
You have to add a relational table `order_products` or `order_items` to accommodate this.